packages/ra-ui-materialui/src/KeyboardShortcut.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { createTheme, ThemeProvider } from '@mui/material';
+import { KeyboardShortcut } from './KeyboardShortcut';
+import { defaultTheme } from './theme';
+
+export default {
+    title: 'ra-ui-materialui/KeyboardShortcut',
+};
+
+const Wrapper = ({ children }: { children: React.ReactNode }) => (
+    <ThemeProvider theme={createTheme(defaultTheme)}>{children}</ThemeProvider>
+);
+
+export const Default = () => (
+    <Wrapper>
+        <KeyboardShortcut keyboardShortcut="meta+K" />
```

Comment:
Could you add some more examples so that we see how they are represented? E.G.:

- ctrl+shift+a+c
- shift+a
- mod+B
- alt+F
- esc+F
- shift+up
- space>a
- 'ctrl+shift+a+c, c, shift+c, alt+n, ctrl+d, meta+d'
- etc.

See https://react-hotkeys-hook.vercel.app/docs/documentation/useHotkeys/basic-usage#modifiers--special-keys for an extensive list.